docs/content/preview/develop/best-practices-ysql.md

```diff
@@ -130,3 +130,20 @@ For more details, see [Distributed parallel queries](../../api/ysql/exprs/func_y
 ## TRUNCATE tables instead of DELETE
 
 [TRUNCATE](../../api/ysql/the-sql-language/statements/ddl_truncate/) deletes the database files that store the table and is much faster than [DELETE](../../api/ysql/the-sql-language/statements/dml_delete/) which inserts a _delete marker_ for each row in transactions that are later removed from storage during compaction runs.
+
+
+## Number of tables
+Each table is split into tablets and each tablet has overhead. See [tablets per server](#tablets-per-server) for limits.
+
+## Tablets per server
+Each table consists of several tablets based on the [`--ysql_num_shards_per_tserver`](../../reference/configuration/yb-tserver#ysql_num_shards_per_tserver)
```

Comment:
`each table` -> `each table and index`
`number of tables` -> `number of tables and indexes`